src/payments/services/premium-certificate-purchase-email.service.ts

```
@@ -0,0 +1,165 @@
+      }
+
+      const intent = await this.paymentIntentService.findById(
+        params.paymentIntentId,
+      );
+
+      const successTransactions = (intent.transactions ?? []).filter(
+        (t) => t.status === PaymentTransactionStatus.SUCCESS,
+      );
+      const latestSuccess = [...successTransactions].sort(
+        (a, b) =>
+          new Date(b.createdAt).getTime() - new Date(a.createdAt).getTime(),
```

ESLint

[error] Delete `,`

(prettier/prettier)